src/qt/bitcoin.cpp

```diff
@@ -259,9 +259,24 @@ void BitcoinApplication::createPaymentServer()
 }
 #endif
 
-void BitcoinApplication::createOptionsModel(bool resetSettings)
+bool BitcoinApplication::createOptionsModel(bool resetSettings)
 {
-    optionsModel = new OptionsModel(node(), this, resetSettings);
+    optionsModel = new OptionsModel(node(), this);
+    if (resetSettings) {
+        optionsModel->Reset();
+    }
+    bilingual_str error;
+    if (!optionsModel->Init(error)) {
+        fs::path settings_path;
+        if (gArgs.GetSettingsPath(&settings_path)) {
+            error += Untranslated("\n");
+            error += strprintf(_("Settings file %s might be corrupt or invalid."), fs::quoted(fs::PathToString(settings_path)));
```

Comment:
Could `_()` be avoided in the GUI code?

See #454.

Reply:
> Could `_()` be avoided in the GUI code?
> 
> See #454.

I'm looking at approach taken in #454, but it doesn't seem to deal with template strings that have parameters. If I need to generalize that approach and use `QObject::tr()` method to format `"Settings file %1 might be corrupt"` template for the GUI, and `strprintf` to format `Settings file %s might be corrupt" template for `debug.log`, that will be make the code more repetitive and make the translated/original code paths diverge more.

I wonder why `_()` should be avoided in GUI code if handles bilingual strings well and we want bilingual strings for an init error? Is there a technical reason why `_()` does not work? Would be be difficult to fix?

Reply:
> I wonder why `_()` should be avoided in GUI code if handles bilingual strings well and we want bilingual strings for an init error? Is there a technical reason why `_()` does not work? Would be be difficult to fix?

The reason is that the `qt` sub-directory is not parsed by the `share/qt/extract_strings_qt.py` script, and it results with missed translatable strings.

Besides #454, there were an alternative approach in bitcoin/bitcoin#22764.